spec/features/clearance/visitor_signs_up_spec.rb

```diff
@@ -1,31 +1,43 @@
 # frozen_string_literal: true
 
-require "rails_helper"
-require "support/features/clearance_helpers"
+require 'rails_helper'
+require 'support/features/clearance_helpers'
 
-RSpec.feature "Visitor signs up" do
-  scenario "by navigating to the page" do
+RSpec.feature 'Visitor signs up' do
+  scenario 'by navigating to the page' do
     visit sign_in_path
 
-    click_link I18n.t("sessions.form.sign_up")
+    click_link I18n.t('sessions.form.sign_up')
 
     expect(current_path).to eq sign_up_path
   end
 
-  scenario "with valid email and password" do
-    sign_up_with "testname", "valid@example.com", "password"
+  scenario 'with valid email and password' do
+    sign_up_with 'testname', 'valid@example.com', 'password'
     visit sign_in_path
     expect_user_to_be_signed_in
   end
 
-  scenario "tries with invalid email" do
-    sign_up_with "testname", "invalid_email", "password"
+  scenario 'with valid email, password, and picture' do
+    user = FactoryBot.build(:user)
+    user.
+      picture.
+      attach(io: File.open('public/apple-touch-icon.png'), filename: 'apple-touch-icon.png')
+
+    visit sign_up_path
+    click_button I18n.t('helpers.submit.user.create')
+
+    expect(user.picture.present?).to eq(true)
```

Comment:
If you want to get fancy:

```suggestion
    expect(user.picture).to be_present
```